weather_app/weather/templates/weather/vista1.html

```diff
@@ -0,0 +1,5 @@
+{% extends 'weather/template.html' %}
+
+{% block mycontent %}
+Hola este es mi vista 1 <br><a href="./pagina2">Ir a pagina 2</a>
```

Comment:
```suggestion
Hola este es mi vista 1 <br><a href="/pagina-prueba">Ir a pagina 2</a>
```
Recuerda que el primer valor del ```path()``` es la URL con la que vas a poder ir buscarlo en el navegador